frontend/src/metabase/querying/filters/components/FilterPicker/FilterColumnPicker/FilterColumnPicker.tsx

```diff
@@ -56,6 +60,9 @@ export function FilterColumnPicker({
   onColumnSelect,
   onSegmentSelect,
   onExpressionSelect,
+  withCustomExpression = true,
+  withColumnGroupIcon = true,
+  withColumnItemIcon = true,
```

Comment:
In the main Metabase app, these are enabled by default. It's not always needed in the SDK though.

Reply:
It looks like this `FilterColumnPicker` component constructs the prop (e.g. `sections`, `renderItem*`) and passes it to `StyledAccordionList` directly, so it's a little difficult to lift it up